src/GroupTypeManager.php

```diff
@@ -191,8 +191,22 @@ public function getGroupBundleIdsByEntityType($entity_type_id) {
    * {@inheritdoc}
    */
   public function getAllGroupBundles($entity_type = NULL) {
+    // Todo - should be remove since this method don't do any thing.
+    return $this->getGroupMap();
+  }
```

Comment:
This change is not correct, it is losing the functionality to filter the group bundles because it is ignoring the `$entity_type` parameter.

It is out of scope for this PR to change existing functionality. Let's only make the changes here that are needed to remove the complex widget.

Reply:
Also see https://github.com/Gizra/og/pull/570/files#r339988596

Reply:
Reverted this change.